frontend/app/views/fragments/event/itemSnapshot.scala.html

```diff
@@ -0,0 +1,21 @@
+@(event: model.Eventbrite.RichEvent, showStats: Boolean = true, showPricing: Boolean = true)
```

Comment:
showStats and showPricing Booleans are not being sent in as false anywhere I can see are these conditionals and arguments needed?